### PR TITLE
Moved path specification to EventDataStream #2956

### PR DESCRIPTION
**Pull request by joachimmetz**
Moved path specification to EventDataStream #2956

### CODE REVIEW

File: plaso/multi_processing/psort.py [hunk 1]
```diff
@@ -84,13 +84,22 @@ def _GetEventIdentifiers(self, event, event_data, event_data_stream):
       event_attributes.extend(event_data_stream.GetAttributes())
 
     for attribute_name, attribute_value in sorted(event_attributes):
-      if attribute_name in self._IDENTIFIER_EXCLUDED_ATTRIBUTES:
+      # The filestat parser operated on file entry level and has no event data
```
Comment (Onager) [inline plaso/multi_processing/psort.py:87]: operates, not operated

File: plaso/multi_processing/psort.py [hunk 2]
```diff
@@ -84,13 +84,22 @@ def _GetEventIdentifiers(self, event, event_data, event_data_stream):
...
+      # stream with a path specification. Therefore we need filename and inode
+      # to make events regarding different file entries are considered unique.
```
Comment (Onager) [inline plaso/multi_processing/psort.py:89]: I think is supposed to read "...to make sure events..."

**Comment by joachimmetz** [comment]
Rebased and addressed merge conflicts.

File: plaso/multi_processing/psort.py (hunk 1 above)
Comment (joachimmetz) [reply plaso/multi_processing/psort.py:87]: done

File: plaso/multi_processing/psort.py (hunk 2 above)
Comment (joachimmetz) [reply plaso/multi_processing/psort.py]: done

**Comment by joachimmetz** [comment]
Addressed review comments.

**Pull request merged**